sdk/protocol-manager-poc/src/implementation/aaveV3Plugin.ts

```diff
@@ -0,0 +1,167 @@
+import { AddressValue, Percentage, TokenAmount, TokenSymbol, Price, CurrencySymbol, RiskRatio } from "@summerfi/sdk-common/common"
+import type { AaveV3LendingPool, AaveV3PoolDebtConfig, AaveV3PoolCollateralConfig, AaveV3PoolId } from "@summerfi/sdk-common/protocols"
+import { PoolType, ProtocolName, EmodeType } from "@summerfi/sdk-common/protocols"
+import { Position, Token } from "@summerfi/sdk-common/common"
+import { BigNumber } from 'bignumber.js'
+import { z } from 'zod'
+import { ChainId, CreateProtocolPlugin, IPositionId, ProtocolManagerContext, ProtocolPlugin } from "../interfaces";
+import { AaveV3LikePluginBuilder, filterAssetsListByEMode } from "./aaveV3LikeBuilder";
+import { UNCAPPED_SUPPLY, PRESISION_BI } from "./constants";
+
+
+function tokenAmountFromBaseUnit({amount, token}: {amount: string, token: Token}): TokenAmount {
```

Comment:
Do we need this function? Seems a bit unnecessary

Reply:
It was entirely unnecessary ;-)